queen-of-versions/build.gradle

```diff
@@ -27,7 +27,7 @@ android {
 }
 
 dependencies {
-    api project(path: ':prince-of-versions')
+    api "co.infinum:prince-of-versions:${versions.prince}"
```

Comment:
Could we live without this?
If we just use this then it would be very hard to develop new version of Queen that requires new version of Prince as we would need to publish Prince artifact to get the changes in Queen.
If this is mandatory, please investigate Gradle dependency substitutions - that might be helpful

Reply:
I made this change in discussion with Bojan. AFAIK both approaches work, just a matter of which one you prefer. I can change it 👍

Reply:
This is a must have, otherwise you end up with duplicate classes errors on compile time because Gradle cannot resolve embedded projects.

Reply:
I have never experienced something like that. Is this something related to Maven publishing?